terraform/modules/gcf_gen2_pubsub_source_repo/main.tf

```diff
@@ -1,167 +1,168 @@
-  project = var.project
-  role    = "roles/secretmanager.secretAccessor"
-  member  = "serviceAccount:${data.google_project.project.number}-compute@developer.gserviceaccount.com"
-}
-
-resource "google_project_iam_member" "token_creator_access" {
-  project = var.project
-  role    = "roles/iam.serviceAccountTokenCreator"
-  member  = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-pubsub.iam.gserviceaccount.com"
-
-  depends_on = [google_project_service.pubsub_api]
-}
-
-resource "google_project_iam_member" "token_creator_access_ce" {
-  project = var.project
-  role    = "roles/iam.serviceAccountTokenCreator"
-  member  = "serviceAccount:${data.google_project.project.number}-compute@developer.gserviceaccount.com"
-}
-
-resource "google_project_iam_member" "run_invoker_access" {
-  project = var.project
-  role    = "roles/run.invoker"
-  member  = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-pubsub.iam.gserviceaccount.com"
-
-  depends_on = [google_project_service.pubsub_api]
-}
-
-resource "google_project_iam_member" "run_invoker_access_ce" {
-  project = var.project
-  role    = "roles/run.invoker"
-  member  = "serviceAccount:${data.google_project.project.number}-compute@developer.gserviceaccount.com"
-}
-
-resource "google_cloudfunctions2_function" "function" {
-  name        = "${var.name}_function"
-  location    = var.region
-  description = var.description
-
-  build_config {
-    runtime               = var.runtime
-    entry_point           = var.entry_point
-    environment_variables = var.environment
-    source {
-      storage_source {
-        bucket = module.source_code.bucket_name
-        object = module.source_code.bucket_object_name
-      }
-    }
-  }
-
-  service_config {
-    max_instance_count             = var.max_instances
-    min_instance_count             = var.min_instances
-    available_memory               = var.available_memory
-    available_cpu                  = var.available_cpu
-    timeout_seconds                = var.timeout
-    environment_variables          = var.environment
-    ingress_settings               = "ALLOW_INTERNAL_ONLY"
-    all_traffic_on_latest_revision = true
-  }
-
-  event_trigger {
-    trigger_region = var.region
-    event_type     = "google.cloud.pubsub.topic.v1.messagePublished"
-    pubsub_topic   = google_pubsub_topic.topic.id
-    retry_policy   = "RETRY_POLICY_DO_NOT_RETRY"
-  }
-
-  depends_on = [
-    google_project_service.cloud_build_api,
-    google_project_service.cloud_functions_api,
-    google_pubsub_topic.topic,
-    google_project_service.artifact_registry_api
-  ]
-}
-
-## alerting policy
-module "alerting_policy" {
-  source = "../alert_policy"
-  count = var.alert_on_failure ? 1 : 0
-  name = "${var.name}-alert-policy"
-  filter = "resource.type=\"cloud_function\" severity=ERROR resource.labels.function_name=\"${var.name}\""
-}
+resource "google_project_service" "cloud_build_api" {

```

Comment:
I don't think there are any changes in this file, but again it signals as if the whole file is replaced, can you check why that is?

I think this PR probably does not mean to change anything here.